bin/kube_overrides.rb

```diff
@@ -37,6 +43,28 @@
         env['valueFrom']['secretKeyRef']['name'] = current_secrets_name
       end
     end
+    overrides.each do |k, v|
+      child = container
+      k[0...-1].each do |elem|
+        child[elem] ||= {}
+        child = child[elem]
+      end
+      case child
+      when Array
+        # Deal with the environment list specially
+        child.reject! do |elem|
+          elem['name'] == k.last
+        end
+        child << {
+          'name'  => k.last,
+          'value' => v,
+        }
+      when Hash
```

Comment:
What is the use case for the hash here?

I think it would be helpful if the comments would include an example of each kind of override.

Reply:
Added example (image pull policy).